src/directive.js

```diff
@@ -68,7 +57,8 @@
                         stoken: scope.stoken || attrs.stoken || null,
                         theme: scope.theme || attrs.theme || null,
                         tabindex: scope.tabindex || attrs.tabindex || null,
-                        size: scope.size || attrs.size || null
+                        size: scope.size || attrs.size || null,
+                        'expired-callback': expired
```

Comment:
I wonder if this should be added as a new argument of [`vcRecaptchaService.create()`](https://github.com/VividCortex/angular-recaptcha/blob/4e8d2744e15c1975a5738ab33442b1ccc1da048c/src/service.js#L60-L67), although it would require a major-version bump due to the change in the api. So far, every time I see this _conf_ object I think of styling settings.